core/Piranha/Services/SiteService.cs

```diff
@@ -465,7 +465,7 @@ private void OnLoadContent(Models.SiteContentBase model)
                     _factory.Init(model, App.SiteTypes.GetById(model.TypeId));
                 }
 
-                App.Hooks.OnLoad<Models.SiteContentBase>(model);
+                App.Hooks.OnLoad(model);

```

Comment:
See comments on PageService and PostService.

Reply:
reverted :)